### PR TITLE
add missing DB dependency to easyconfig for Perl 5.32.0

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

Same as #11451

### CODE REVIEW

**Comment by branfosj** [comment]
Marking this PR as a draft while I debug a problem with the install:
```
Installing documentation: /rds/bear-apps/devel/eb-sjb-up/EL7/EL7-cascadelake/software/DB/18.1.40-GCCcore-10.2.0/docs ...
cp: cannot stat bdb-sql: No such file or directory
cp: cannot stat gsg_db_server: No such file or directory
```

**Comment by branfosj** [comment]
And, on checking neither of `bdb-sql` nor `gsg_db_server` appear in the `docs` folder:
```
$ ls db-18.1.40/docs/
api_reference  articles  collections  csharp  gsg  gsg_db_rep  gsg_txn  index.html  installation  java  license  porting  programmer_reference  upgrading
```

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/63fd1063e139d70812fd83f72555ce1e for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bear-pg0206u16a.bear.cluster - Linux centos linux 8.2.2004, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/7134febe4de8778674eb476a0862acf0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
node3406.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/2ca64ff7550d249f8de974df882df15c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
node2601.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/3625123377155de561317dad3c3f491e for a full test report.

File: easybuild/easyconfigs/d/DB/DB-18.1.40-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+description = """Berkeley DB enables the development of custom data management
+ solutions, without the overhead traditionally associated with such custom
+ projects."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+# download via
+# https://www.oracle.com/technetwork/products/berkeleydb/downloads/,
+# requires registration
```
Comment (boegel) [inline easybuild/easyconfigs/d/DB/DB-18.1.40-GCCcore-10.2.0.eb:15]: @branfosj We can actually auto-download this via `https://download.oracle.com/berkeley-db/db-18.1.40.tar.gz`.

Only works for latest version though, it seems...

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/d/DB/DB-18.1.40-GCCcore-10.2.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/d/DB/DB-18.1.40-GCCcore-10.2.0.eb:15]: Better option could be this Homebrew mirror, it does rename the source tarball though (but SHA256 checks out):

```
https://bintray.com/homebrew/mirror/download_file?file_path=berkeley-db-18.1.40.tar.gz
```

Comment (boegel) [reply easybuild/easyconfigs/d/DB/DB-18.1.40-GCCcore-10.2.0.eb:15]: see also #11454

Comment (branfosj) [reply easybuild/easyconfigs/d/DB/DB-18.1.40-GCCcore-10.2.0.eb]: Yes, switching to the Homebrew mirror and having an automatic download is sensible to me. I've done that in d62b2a3.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/7de9e852f43261ed5785a4cee14f0096 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3404.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/7eb001ac4fa6fc1e2dfa7ee37ce38fc3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/54246c32737cd8e5516ace8db353471e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2627.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/f4c423527b829a08906187129496459c for a full test report.

**Comment by boegel** [comment]
Going in, thanks @branfosj!

**Pull request merged**